test/x/client_initialization_test.rb

```diff
@@ -2,13 +2,9 @@
 require_relative "../test_helper"
 
 module X
-  class ClientInitializationTest < Minitest::Test
+  class ClientOAuthInitializationTest < Minitest::Test
```

Comment:
Why this change ? It's just for a better structure ?

Reply:
Yeah, I wanted to group tests in this file into the following classes:
* `ClientOAuthInitializationTest`
* `ClientOAuth2InitializationTest`
* `ClientAuthenticatorPrecedenceTest`
* `ClientConnectionOptionsTest`
* `ClientDefaultsTest`